library/src/include/rocsparse_assign_async.hpp

```diff
@@ -0,0 +1,33 @@
+/*! \file */
+/* ************************************************************************
+ * Copyright (C) 2018-2025 Advanced Micro Devices, Inc. All rights Reserved.
```

Comment:
Wrong date for new file?

Reply:
Good question. Some of the content of the file comes from `utility.hpp` (for example `assign_async`) that has an initial date of 2018. The file is new but the content is old. I suppose that using 2018 is more conservative here.

Reply:
@doctorcolinsmith advised to put 2025.

Reply:
Fixed